### DISCUSSION TITLE
Inquiring On The Topics Of Card Customization(s) And Attribution(s)

### DISCUSSION THREAD

**Discussion by dylanhood** [DenverCoder1/github-readme-streak-stats#486] [Q&A] [answered]
How, exactly, am I supposed to attribute the author(s) of this repository? Am I allowed to add the attribution(s) to a file in my repository called "Attributions.txt"?

If I donate does that mean I no longer have to add an attribution?

How do I get the card to only display contributions?

I tried something like this (along with trying other stuff)

 <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://streak-stats.demolab.com?user=dylanhood&$currentStreak=hidden&$longestStreak=hidden&$currStreakNum=hidden&$fire=hidden&$ring=hidden" />
    <img src="https://streak-stats.demolab.com?user=dylanhood&theme=git-dark&hide_currentStreak=true&hide_longestStreak=true&hcurrStreakNum=false&fire=false&ring=false" />
</picture>

How do I remove the "Current Streak" and "Longest Streak" sections from the card?

**Comment by DenverCoder1** [comment]
Attribution is not required although it's appreciated. Making the image link back to the repository is recommended but you can also attribute it in any way you like.

~There is not currently an option to hide sections of the card, but if you want to show the stat and not the others, you could try cards from other repos such as [anuraghazra/github-readme-stats](https://github.com/anuraghazra/github-readme-stats) or [vn7n24fzkq/github-profile-summary-cards](https://github.com/vn7n24fzkq/github-profile-summary-cards). There are good lists of widgets such as [rzashakeri/beautify-github-profile](https://github.com/rzashakeri/beautify-github-profile).~ The option now exists, see other comment on discussion.

You can also fork the repo to modify the card or use the json API to create something, for example, a badge like below.

[![badge](https://custom-icon-badges.herokuapp.com/badge/dynamic/json?logo=graph&logoColor=fff&color=blue&label=total%20contributions&query=%24.totalContributions&url=https%3A%2F%2Fgithub-readme-streak-stats.herokuapp.com%2F%3Fuser%3DDenverCoder1%26type%3Djson)](https://custom-icon-badges.herokuapp.com/badge/dynamic/json?logo=graph&logoColor=fff&color=blue&label=total%20contributions&query=%24.totalContributions&url=https%3A%2F%2Fgithub-readme-streak-stats.herokuapp.com%2F%3Fuser%3DDenverCoder1%26type%3Djson)

https://custom-icon-badges.herokuapp.com/badge/dynamic/json?logo=graph&logoColor=fff&color=blue&label=total%20contributions&query=%24.totalContributions&url=https%3A%2F%2Fgithub-readme-streak-stats.herokuapp.com%2F%3Fuser%3DDenverCoder1%26type%3Djson

**Comment by DenverCoder1** [comment, marked as answer]
> How do I remove the "Current Streak" and "Longest Streak" sections from the card?

You can now remove sections of the card using the following options:

|         Parameter          |                     Details                      |                                              Example                                               |
| :------------------------: | :----------------------------------------------: | :------------------------------------------------------------------------------------------------: |
| `hide_total_contributions` | Hide the total contributions (Default: `false`)  |                                         `true` or `false`                                          |
|   `hide_current_streak`    |    Hide the current streak (Default: `false`)    |                                         `true` or `false`                                          |
|   `hide_longest_streak`    |    Hide the longest streak (Default: `false`)    |                                         `true` or `false`                                          |

The `card_width` option can also be used to remove some of the extra padding.

```md
https://streak-stats.demolab.com?user=dylanhood&theme=git-dark&hide_longest_streak=true&hide_current_streak=true&card_width=200
```

![](https://streak-stats.demolab.com?user=dylanhood&theme=git-dark&hide_longest_streak=true&hide_current_streak=true&card_width=200)

**Reply by Mcjizabaw** [reply]
Who exactly does this involve

**Reply by Mcjizabaw** [reply]
I don't know I just came across all of this information I didn't even know that I had an organization set up I wasn't aware of any activity in an organization that was founded by me falsely I'm just as clueless as you may be are I can't find a way to understand who would do this when all I did was take the very first step in trying to set up a nonprofit organization for homeless women and children to get clothes and food because I was homeless and I don't know what it's like.
